prepare_vscode.sh

```diff
@@ -13,6 +13,14 @@ patch -u src/vs/platform/update/electron-main/updateService.win32.ts -i ../patch
 
 yarn --frozen-lockfile
 yarn postinstall
+
+if [[ "$BUILDARCH" == *"arm"* ]]; then
+  sed -i -z 's/,\n[^\n]*arm[^\n]*//' node_modules/vscode-sqlite3/binding.gyp
+  sed -i "s/Release\/sqlite'/Release\/sqlite.node'/" node_modules/vscode-sqlite3/lib/sqlite3.js
+  yarn add -D electron-rebuild
+  npx electron-rebuild -f -w vscode-sqlite3
```

Comment:
if using `npx` there should be no reason to save `electron-rebuild` as a devDep on L20

Reply:
Do you mean we can omit this line?  `yarn add -D electron-rebuild`

Reply:
yes

Reply:
I don't think removing that line works. I got this error when trying to use `npx` only
```
Step 29/36 : RUN npx electron-rebuild -f -w vscode-sqlite3
 ---> Running in d8195f16eb1b
npx: installed 164 in 38.491s

An unhandled error occurred inside electron-rebuild
Unable to find electron's version number, either install it or specify an explicit version

Error: Unable to find electron's version number, either install it or specify an explicit version
    at /root/.npm/_npx/1/lib/node_modules/electron-rebuild/lib/src/cli.js:87:19
    at Generator.next (<anonymous>)
    at fulfilled (/root/.npm/_npx/1/lib/node_modules/electron-rebuild/lib/src/cli.js:6:58)
```

Reply:
I see... I guess electron-rebuild does some kind of install-time logic. LGTM